compute/cker/include/cker/operation/LogicalOr.h

```diff
@@ -62,12 +62,20 @@ inline void LogicalOrBroadcast4D(const Shape &unextended_input1_shape, const T *
 }
 
 template <typename T>
-inline void LogicalOr(const Shape &unextended_input1_shape, const T *input1_data,
-                      const Shape &unextended_input2_shape, const T *input2_data,
-                      const Shape &unextended_output_shape, T *output_data)
+inline void LogicalOrElementwise(const Shape &shape, const T *input1_data, const T *input2_data,
+                                 T *output_data)
 {
-  LogicalOrBroadcast4D<T>(unextended_input1_shape, input1_data, unextended_input2_shape,
-                          input2_data, unextended_output_shape, output_data);
+  int num_elements = 1;
+
+  for (int t = 0; t < shape.DimensionsCount(); t++)
+  {
+    num_elements *= shape.Dims(t);
+  }
+
+  for (int t = 0; t < num_elements; t++)
+  {
+    *(output_data + t) = *(input1_data + t) || *(input2_data + t);
```

Comment:
```suggestion
    output_data[t] = input1_data[t] || input2_data[t];
```
How about changing this code as array access?

Reply:
array access makes the code look clean, hence incorporated.